internal/pkg/ntp/internal/spike/spike.go

```diff
@@ -74,7 +74,8 @@ func (d *Detector) IsSpike(sample Sample) bool {
 	var j float64
 
 	for i := range d.samples {
-		j += math.Pow(d.samples[i].Offset-d.samples[indexMin].Offset, 2)
+		offsetDiff := d.samples[i].Offset - d.samples[indexMin].Offset
+		j += offsetDiff * offsetDiff
```

Comment:
nit: Wouldn't `math.Pow` look better here?

Reply:
the linter thinks unrolled version would work faster, which I think is true

Reply:
Valid unless the compiler unrolls it too

Reply:
I can't find any proof that it'd be unrolled, at least it's not a special case in the function itself.